### DISCUSSION TITLE
0 주차

### DISCUSSION THREAD

**Discussion by sunhpark42** [elegant-functional-2023/javascript-sicp-2023#2] [General]
스터디를 시작하기 전에 바라는 것, 얻어가고 싶은 것을 한마디씩 적어주세요 🤩

**Comment by hanana1253** [comment]
- SICP 책을 함께 완독하는 것
- 컴퓨터 프로그래밍과 함수형에 대하여 다차원적으로 이해하고 정의내릴 수 있으면 좋겠다

**Comment by thgr8ganzi** [comment]
도전, 책이 상당히 어려운데.. 😢
이런 고전은 도전하는것 자체에 의미가 있다고 생각합니다. 책을 이해 하진 못하더라도 간단히 훝고 나중에 기회가 되면 다세 읽어보고 싶습니다.

**Comment by sunhpark42** [comment]
절대 혼자 하지 않을 컴퓨터 기반 지식에 대한 공부,
그리고 내가 사용 하고 있는 컴퓨터에 대한 구조적인 이해가 하고 싶어요😎

**Comment by ddongule** [comment]
바라는 것
- 다같이 즐겁게 책 조지기!

얻어가고 싶은 것
- 컴퓨터의 구조에 대해 전반적으로 이해할 수 있으면 좋겠다! (이해까지 아니더라도 감이 잡히는 정도만 돼도 기쁠듯^^..)

**Reply by sunhpark42** [reply]
![image](https://github.com/elegant-functional-2023/javascript-sicp-2023/assets/67677561/0de23e20-2864-4115-9020-112c4f6aa410)

**Comment by chloe-codes1** [comment]
- 기반이 되는 CS 지식과 프로그래밍 언어의 동작 원리에 대한 이해도
- 스터디원들과의 의견 교환을 통한 책 내용 그 이상의 지식 습득

**Comment by iborymagic** [comment]
1. 이 책을 반 이상 읽었을 정도면 이제 다른 두꺼운 서적들은 두렵지 않을거라는 자신감
2. 컴퓨터 내부 동작에 대한 적당한 이해

**Comment by sa02045** [comment]
- 아직 이 책이 무슨 내용의 책인지 감이 잡히지 않는데, 다 읽고나서 이런 책이었구나하고 이해하기
- 책과 함께 컴퓨터에 대한 근본을 공부하기

**Comment by bigsaigon333** [comment]
- 대규모 소프트웨어 시스템의 지적인 복잡성을 제어하는데 사용하는 기법들을 익힌다
- 사람들이 읽을 것을 염두에 두고 프로그램을 작성하는 방법을 익힌다
- `하나의 자료 구조에 작동하는 함수 100개를 두는 것이 10개의 자료 구조에 작동하는 함수 10개를 두는 것보다 낫다` 를 나의 언어로 설명할 수 있다

**Comment by yunseorim1116** [comment]
공부를 할 때 내가 가진 한계점 보다 더 높은 지점을 바라보며 공부를 해야 한계점을 높일 수 있다고 생각합니다. 나에게만 어려운 책이 아닌, 모두에게 어려운 책을 읽고 함께 토론해보며 지속적으로 공부하고 싶습니다. 추가적으로 이 책을 통해 컴퓨터 용어에 관련해 지식을 찾아본다면 관련 지식 이해도를 더 효과적으로 높일 수 있을 것이라 기대합니다!

**Reply by bigsaigon333** [reply]
서림님 돌아와~~

**Reply by yunseorim1116** [reply]
보고싶어요 여러부우운 😭😭😭
